### ISSUE TITLE
[BUG]can't access to the dashboard with 'Handler disconnected'

### ISSUE THREAD

**Issue by szthanatos** [harvester/harvester#1758]
**Describe the bug**
After using harvester for a while, I can't access to the webpage anymore.
![image](https://user-images.githubusercontent.com/3314301/147738204-3b169ded-52fb-41e3-b76b-d14f33cb907f.png)

**To Reproduce**
The last 2 thing I do was upload image at background & create 2 VM.

When I push the create button, it failed with `Handler disconnected`, then the while page become unaccessable.

**Environment:**
 - Harvester ISO version:  V1.0.0
 - Underlying Infrastructure: Dell OptiPlex 5060

**Comment by szthanatos** [comment]
the only page I can goto is:
![image](https://user-images.githubusercontent.com/3314301/147739011-0df01c92-c804-4476-a925-a473cbf5f086.png)

use `kubectl` found all deployment, svc, pod running.

pod rancher's latest log as this:
```logs
2021/12/30 07:49:20 [INFO] Downloading repo index from http://harvester-cluster-repo.cattle-system/charts/index.yaml
W1230 07:53:43.745225      33 transport.go:288] Unable to cancel request for *client.addQuery
W1230 07:53:44.311304      33 transport.go:288] Unable to cancel request for *client.addQuery
W1230 07:53:44.319714      33 transport.go:288] Unable to cancel request for *client.addQuery
W1230 07:53:44.319783      33 transport.go:288] Unable to cancel request for *client.addQuery
W1230 07:53:44.321527      33 transport.go:288] Unable to cancel request for *client.addQuery
2021/12/30 07:54:20 [INFO] Downloading repo index from http://harvester-cluster-repo.cattle-system/charts/index.yaml
2021/12/30 07:54:33 [INFO] error in remotedialer server [400]: write tcp 10.52.0.219:444->10.52.0.187:37422: i/o timeout
E1230 07:54:33.711474      33 upgradeaware.go:387] Error proxying data from client to backend: write tcp 10.52.0.219:444->10.52.0.187:37422: i/o timeout
E1230 07:54:33.711548      33 upgradeaware.go:401] Error proxying data from backend to client: readfrom tcp 10.52.0.219:80->172.16.70.179:47136: tunnel disconnect
E1230 07:54:33.711551      33 upgradeaware.go:401] Error proxying data from backend to client: readfrom tcp 10.52.0.219:80->172.16.70.179:42706: tunnel disconnect
E1230 07:54:33.711573      33 upgradeaware.go:387] Error proxying data from client to backend: write tcp 10.52.0.219:444->10.52.0.187:37422: i/o timeout
E1230 07:54:33.711560      33 upgradeaware.go:387] Error proxying data from client to backend: write tcp 10.52.0.219:444->10.52.0.187:37422: i/o timeout
E1230 07:54:33.711588      33 upgradeaware.go:401] Error proxying data from backend to client: readfrom tcp 10.52.0.219:80->172.16.70.179:36620: tunnel disconnect
2021/12/30 07:58:12 [ERROR] Error during subscribe websocket: close sent
2021/12/30 07:59:01 [ERROR] Error during subscribe websocket: close sent
2021/12/30 07:59:20 [INFO] Downloading repo index from http://harvester-cluster-repo.cattle-system/charts/index.yaml
2021/12/30 08:02:04 [ERROR] Error during subscribe websocket: close sent
2021/12/30 08:04:20 [INFO] Downloading repo index from http://harvester-cluster-repo.cattle-system/charts/index.yaml
2021/12/30 08:09:20 [INFO] Downloading repo index from http://harvester-cluster-repo.cattle-system/charts/index.yaml
2021/12/30 08:14:20 [INFO] Downloading repo index from http://harvester-cluster-repo.cattle-system/charts/index.yaml
2021/12/30 08:19:20 [INFO] Downloading repo index from http://harvester-cluster-repo.cattle-system/charts/index.yaml
```

**Comment by guangbochen** [comment]
can u also help to grasp the log of the harvester pod? since only one will be the master pod so u will need to check all 3 pods respectively.

**Comment by ibayue** [comment]
time="2022-01-18T11:50:42Z" level=error msg="Error writing ping" error="write tcp 10.52.0.84:43744->10.53.54.220:443: i/o timeout"
10.53.54.220:443看了下是rancher pod的svc，手动在harvester pod里curl 10.53.54.220:443正常返回没有超时，删除harvester pod重启后恢复正常

**Comment by emilyastranova** [comment]
I have run into the same problem during the cancellation of an ISO upload

**Comment by cybaware** [comment]
> I have run into the same problem during the cancellation of an ISO upload

same for me...

**Comment by anandm19** [comment]
Im also facing the same issue and I did cancelled an ISO upload
Is there a workaround yet, pls advise

**Comment by w13915984028** [comment]
It is highly possible due to a known issue(#1904,  #1956), is fixed in v1.0.1

A workaround in versions before v1.0.1, please try:
delete the harvester pod via kubectl, wait one minute, force refreshing the WebUI
like: `kubectl delete pod harvester-5c87b6db68-xnkbt -n harvester-system`

**Comment by w13915984028** [comment]
@szthanatos Please try the latest Harvester.
https://github.com/harvester/harvester/releases/tag/v1.0.2

There is an known issue and is fixed in v1.0.1.  A fast image uploading may cause OOM.

**Comment by w13915984028** [comment]
Close. 

In case it happens again, feel free to reopen. Thank.
